include/opentxs-proto/verify/VerifyBlockchain.hpp

```diff
@@ -59,6 +64,18 @@ static const VersionMap Bip44AccountAllowedBip44Address = {
 static const VersionMap Bip44AccountAllowedHDPath = {
     {1, {1, 1}},
 };
+static const VersionMap Bip47AddressAllowed = {
```

Comment:
The format for these variables is "FooAllowedBar" where "Foo" is the parent class and "Bar" is the embedded child class.

Reply:
Changed